indico/modules/events/surveys/templates/manage_questionnaire.html

```diff
@@ -1,35 +1,82 @@
 {% extends 'events/surveys/manage_survey.html' %}
+{% from 'forms/_form.html' import form_row %}
 
 {% block content %}
     <div class="toolbar">
         <div class="group">
-            {% for field_type in field_types %}
+            {% for field_type in field_types.iterkeys() %}
+                {% set friendly_name = field_types[field_type].friendly_name %}
                 <a href="#" data-href="{{ url_for('.add_question', survey, type=field_type) }}"
-                   data-title="{% trans %}Add {{ field_type }} question{% endtrans %}"
-                   class="i-button js-question-dialog">{% trans %}Add {{ field_type }}{% endtrans %}</a>
+                   data-title="{% trans %}Add {{ friendly_name }} question{% endtrans %}"
+                   class="i-button js-question-dialog">
+                        {% trans friendly_name=friendly_name %}
+                            Add {{ friendly_name }}
+                        {% endtrans %}
+                </a>
             {% endfor %}
         </div>
     </div>
-    <div class="i-box titled no-padding">
-        <div class="i-box-header">
+
+    <div id="questions" class="i-box titled">
+         <div class="i-box-header">
             <div class="i-box-title">
-            {% trans %}List of questions{% endtrans %}
+                {% trans %}Questionnaire{% endtrans %}
             </div>
         </div>
-        <div class="i-box-content">
-            <ul class="group-list with-buttons no-content-before">
-            {% for question in survey.questions %}
-                <li>
-                    <a href="#" data-href="{{ url_for('.edit_question', question) }}"
-                                data-title="{% trans %}Edit question{% endtrans %}"
-                                class="js-question-dialog">
-                        <strong>{{ question.title }}</strong>
-                    </a>
-                </li>
-            {% else %}
-                <li>{% trans %}No questions defined{% endtrans %}</li>
-            {% endfor %}
+        <div class="sortblock i-box-content" style="width: 100%">
+            <ul class="group-list no-content-before">
+                {% for question in survey.questions %}
+                    <li>
+                        <div class="questions-handle i-form">
+                            <div class="question-list">
+                                <div class="question-row">
+                                    <div class="question-data position-td" style="width: 5%;">{{ loop.index }}.</div>
+                                    <div class="question-data">{{ form_row(preview_form['question_{}'.format(question.id)],
```

Comment:
why not start the `{{` in a new line? that way you don't have to wrap the call so much...

```
                                    <div class="question-data">
                                        {{ form_row(...) }}
                                    </div>
```